data/sheriffOladimeji.json

```diff
@@ -0,0 +1,19 @@
+{
+  "name": "Sheriff Oladimeji",
+  "displayStatsPublic": true,
```

Comment:
We no longer need the `displayStatsPublic` key. So please remove this field from your json.

```suggestion
```